tenzir/tests/exec/ocsf/apply.tql

```diff
@@ -0,0 +1,8 @@
+from(
+  {},
+  {class_uid: ""},
+  {class_uid: 42},
+  {class_uid: 1002},
+  {class_uid: 4002, count: 42, nmapped: {foo: 192.168.0.1}},
```

Comment:
```suggestion
  {class_uid: 4002, count: 42, unmapped: {foo: 192.168.0.1}},
```

This shows that you like should warn for excess fields.

Reply:
Definitely. Same with type mismatches. It's just an experiment, not production ready. I can also do this (and fix this), but I won't push anything now and let CI go through so that we get a Docker build out of it.